airbyte-integrations/connectors/source-looker/source_looker/spec.json

```diff
@@ -9,9 +9,9 @@
     "properties": {
       "domain": {
         "type": "string",
-        "pattern": ["^[a-zA-Z0-9._-]*\\.looker\\.com$"],
-        "examples": ["domainname.looker.com"],
-        "description": "Domain for your Looker account, e.g. airbyte.cloud.looker.com"
+        "pattern": ["(^[a-zA-Z0-9._-]*\\.looker\\.com$)|(^looker\\.[a-zA-Z0-9._-]*\\.com$)|(^[0-9]+(?:\\.[0-9]+){3}(:[0-9]+)?$)"],
```

Comment:
Why should the second pattern have looker in the name? It should just be any url right?

Reply:
I see looker.[yourdomain].com  in the documentation.
https://docs.looker.com/setup-and-management/on-prem-install/installation#create_a_dns_record

If that's not the case , ill change to support any url

Reply:
I'm guessing that's more of a suggestion than a requirement. We can probably give that pattern in the examples array but not strictly require it.

Reply:
Also we shouldn't require `.com`

Reply:
could you give me some examples so that i get clarity on how to write regex pattern